plugins/gitopen.lua

```diff
@@ -0,0 +1,53 @@
+-- mod-version:3
+local core = require "core"
+local command = require "core.command"
+local common = require "core.common"
+
+
+local function exec(cmd)
+  local proc = process.start(cmd)
+  while proc:running() do
+    coroutine.yield(0.1)
+  end
+  if proc:returncode() > 0 then
+    core.error("ERROR - command: " .. table.concat(cmd, " "))
+  end
+  return proc:read_stdout() or ""
+end
+
+
+local function git_find_files_and_open(commit)
+  local git_root = exec({"git", "rev-parse", "--show-toplevel"}):match( "^%s*(.-)%s*$" )
+  local file_list_str = exec({"git", "show", "--name-only", "--pretty=format:", commit})
+
+  local git_files = {}
+  for str in string.gmatch(file_list_str, "([^\n]+)") do
+    git_files[git_root .. PATHSEP .. str] = true
+  end
+
+  for dir, item in core.get_project_files() do
+    local key = dir .. PATHSEP .. item.filename
+    if git_files[key] then
+      core.root_view:open_doc(core.open_doc(item.filename))
+    end
+  end
+end
+
+command.add("core.docview", {
```

Comment:
Wouldn't it be better to allow calling the command even if the user isn't in a `DocView`?